ZeuZ_Node_Installer.py

```diff
@@ -163,7 +163,7 @@ def get_required_mods():
             else:
                 print("Error - Need root privleges\n")
                 quit()
-
+            print("Asche")
             install(type="sudo", cmd="easy_install pip")
```

Comment:
Remove debuging print

Reply:
Done removing

Reply:
Dont merge it. Its not been tested. Just solved all the issues on my laptop so just saved the codes in separate branch